crates/telemetry/src/env.rs

```diff
@@ -37,6 +38,15 @@ pub(crate) fn otel_metrics_enabled() -> bool {
     ]) && !otel_sdk_disabled()
 }
 
+/// Returns a boolean indicating if the compatibility layer that emits tracing events from
+/// applications logs should be disabled.
```

Comment:
This feels like it is going to end with me staring at an `if` expression with my head turned 90 degrees going "so if it's _not_ disabled then it's enabled and so in the else clause it's _not_ not disabled which wait" like some unnerving game of Boolean RoboRally.  Consider flipping this to be the positive case?  I guess it depends on usage though.

Reply:
I originally had it as the positive case b/c I too wanted to avoid Boolean RoboRally. However, where I was actually using it was `!enabled` so I figured I should just avoid double negatives.

Reply:
Yes, sorry, I had meant to come back to this comment after seeing where it was used.  I no longer have strong feelings about this.  (Feelings, maybe.  But not strong ones!)